android/app/src/main/java/com/microsoft/codepush/common/managers/CodePushUpdateManager.java

```diff
@@ -395,10 +395,10 @@ public void unzipPackage(File downloadFile) throws CodePushUnzipException {
         String unzippedFolderPath = getUnzippedFolderPath();
         try {
             mFileUtils.unzipFile(downloadFile, new File(unzippedFolderPath));
+            mFileUtils.deleteFileOrFolderSilently(downloadFile);
```

Comment:
What is the reason for moving it here? This method isn't throw no more exception anyway.

Reply:
@ruslan-bikkinin now it throws an `IOException`